cmd/utils.go

```diff
@@ -2016,6 +2017,54 @@ func Targz(source, target string) error {
 		})
 }
 
+//Compares the minimum requirements of a stack against the user to determine whether they can use the stack or not.
+func CheckStackRequirements(requirementArray map[string]string, buildah bool) error {
+	versionRegex := regexp.MustCompile(`(\d)+\.(\d)+\.(\d)+`)
+	upgradesRequired := 0
+
+	Info.log("Checking stack requirements...")
+
+	for technology, minVersion := range requirementArray {
```

Comment:
As @kylegc mentioned, in some development scenarios, `appsody version` may return `vlatest`.
There are also situations where we return `0.0.0` (because unfortunately the debian and RPM packages can't be labelled `vlatest`).
 I would suggest checking these specific conditions and warning the user that there are no guarantees that the stack will work with this development version of Appsody.

Reply:
Thanks for the review @chilanti - I've added a check so that if a valid version specified by the regex can't be retrieved from the cmd output (or if the version is 0.0.0), the user will be warned.